packages/docmanager/src/recents.ts

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { PageConfig } from '@jupyterlab/coreutils';
+import { Contents, ServerConnection } from '@jupyterlab/services';
+import { IStateDB } from '@jupyterlab/statedb';
+import { IDisposable } from '@lumino/disposable';
+import { Debouncer } from '@lumino/polling';
+import { ISignal, Signal } from '@lumino/signaling';
+import { IRecentsManager, RecentDocument } from './tokens';
+
+/**
+ * Manager for recently opened and closed documents.
+ */
+export class RecentsManager implements IRecentsManager, IDisposable {
```

Comment:
I guess this is not needed any longer, isn't it?

```suggestion
export class RecentsManager implements IRecentsManager {
```